src/Analyzers/CSharp/Analysis/RemovePartialModifierFromTypeWithSinglePartAnalyzer.cs

```diff
@@ -12,7 +13,13 @@ namespace Roslynator.CSharp.Analysis;
 public sealed class RemovePartialModifierFromTypeWithSinglePartAnalyzer : BaseDiagnosticAnalyzer
 {
     private static ImmutableArray<DiagnosticDescriptor> _supportedDiagnostics;
-    private static readonly MetadataName _componentBaseName = MetadataName.Parse("Microsoft.AspNetCore.Components.ComponentBase");
+
+    private static readonly MetadataName[] _componentBaseNames = new MetadataName[] {
```

Comment:
Can you change field name to something else? _metadataNames would be probably fine.

Reply:
Done!